src/tokens.js

```diff
@@ -822,7 +829,19 @@ export function trackVariables(context = {}) {
       }
 
       if (term === ContextEntry) {
-        return wrap(variables, 'ContextEntry', code);
+        const parts = variables.children.filter(c => c.name !== 'ContextEntry');
+
+        const name = parts[0];
+        const value = parts[Math.max(1, parts.length - 1)];
```

Comment:
Is there a particular reason this is necessary, i.e. do we assume a context entry to be composed of less than two entries at any point?

Reply:
I thought this was necessary for correct error handling (e.g. `{ foo: }`), but I just confirmed that this also has 2 entries. I'll change it

Reply:
You may want to add a dedicated test case for :arrow_up:, too.

Reply:
simplified in [6f9f031](https://github.com/nikku/lezer-feel/pull/16/commits/6f9f0316787d66b1093820456e4c4dc66aa97702) and tested in [304ddab](https://github.com/nikku/lezer-feel/pull/16/commits/304ddab9ae9c2b2eb4df7bca15dd8940e5527e13)